### PR TITLE
chore(consensus): [CON-1341] Inline `adding_nodes_to_subnet_test` and `node_reassignment_test` system tests

### PR DESCRIPTION
**Pull request by kpop-dfinity**

### CODE REVIEW

**Review by dist1ll — APPROVED** [review]

File: rs/tests/Cargo.toml [hunk 1]
```diff
@@ -590,6 +590,10 @@ path = "testing_verification/testnets/large.rs"
 name = "ic-systest-consensus-performance"
 path = "consensus/consensus_performance.rs"
 
+[[bin]]
+name = "ic-systest-adding-nodes-to-subnet-test"
+path = "consensus/adding_nodes_to_subnet_test.rs"
+
```
Comment (basvandijk) [inline rs/tests/Cargo.toml:596]: What about moving this to a `Cargo.toml` to `rs/tests/consensus`?

**Comment by kpop-dfinity** [comment]
Thanks, Adrian

File: rs/tests/Cargo.toml (hunk 1 above)
Comment (kpop-dfinity) [reply rs/tests/Cargo.toml:596]: Good idea, let me take a look

Comment (kpop-dfinity) [reply rs/tests/Cargo.toml]: PTAL @basvandijk

**Review by basvandijk — APPROVED** [review]
Thanks for doing this! LGTM although the empty dependencies of the `cup_compatibility_test` seem a bit suspicious to me.

**Comment by kpop-dfinity** [comment]
> Thanks for doing this! LGTM although the empty dependencies of the `cup_compatibility_test` seem a bit suspicious to me.

Oups :D I forgot to fix that

**Comment by basvandijk** [comment]
Yeah you probably at least want `//rs/tests/driver:ic-system-test-driver` there.

**Comment by kpop-dfinity** [comment]
> Yeah you probably at least want `//rs/tests/driver:ic-system-test-driver` there.

Yep, there were a couple more deps. I hope it's going to compile now

**Pull request merged**